OSDOCS-16264#adding AWS EUSC (#110585)
openshift/openshift-docs: modules/installation-aws-regions.adoc

* `us-west-1` (N. California)
* `us-west-2` (Oregon)

[id="installation-aws-eusc_region_{context}"]
== {aws-short} EUSC region
:FeatureName: European Sovereign Cloud (EUSC) region
include::snippets/technology-preview.adoc[leveloffset=+1]

Installing an {product-title} cluster into the {aws-short} European Sovereign Cloud (EUSC) region helps maximize the sovereignty of your data and satisfy your organization's regulatory requirements. The {aws-short} EUSC is separate and independent from other {aws-short} regions. The infrastructure is located wholly within the European Union (EU). For more information, see https://aws.amazon.com/blogs/security/establishing-a-european-trust-service-provider-for-the-aws-european-sovereign-cloud/[Establishing a European trust service provider for the {aws-short} European Sovereign Cloud] in the {aws-short} documentation.

The following {aws-short} EUSC region is supported:

* `eusc-de-east-1` (Brandenburg)

[IMPORTANT]
====

> 

Reviewer: ```suggestion
```

The following list outlines the limitations that apply to installing an {product-title} cluster into the {aws-short} EUSC region:

* Only one region, `eusc-de-east-1`, and two zones in that region are available.

* The Amazon Machine Images (AMIs) for public {op-system-first} are not yet available in the EUSC region. As a workaround, until the AMI publication for {op-system} is extended to the EUSC region, you must edit your `install-config.yaml` file to specify a custom AMI in the `amiID` field.

* Support is not yet provided for the {aws-short} Security Token Service (STS).

* Support is not yet provided for installing a cluster into a shared Virtual Private Cloud (VPC) with a cross-account private hosted zone.

> 

Reviewer: ```suggestion
```
not a merge blocker, but i have no idea if dita will trip on extraneous line spaces in admonitions; something to consider

* `us-west-1` (N. California)
* `us-west-2` (Oregon)

> [id="installation-aws-eusc_region{context}"]

Reviewer 2: ```suggestion
[id="installation-aws-eusc_region_{context}"]
```

Author: Applied this change.

== {aws-short} EUSC region
:FeatureName: European Sovereign Cloud (EUSC) region
include::snippets/technology-preview.adoc[leveloffset=+1]

> Installing an {product-title} cluster into the {aws-short} European Sovereign Cloud (EUSC) region helps maximize the sovereignty of your data and satisfy your organization's regulatory requirements. The {aws-short} EUSC is separate and independent from other {aws-short} regions. The infrastructure is located wholly within the European Union (EU). For more information, see "Establishing a European trust service provider for the {aws-short} European Sovereign Cloud".

Reviewer 3: The section "Establishing a European trust service provider for the {aws-short} European Sovereign Cloud" is supposed to be a hyperlink, right? It seems to be plain text to me (i.e. no link to aws annoucement)...

The following {aws-short} EUSC region is supported:

* `eusc-de-east-1` (Brandenburg)

[IMPORTANT]
====

The following list outlines the limitations that apply to installing an {product-title} cluster into the {aws-short} EUSC region:

* Only one region, `eusc-de-east-1`, and two zones in that region are available.

* The Amazon Machine Images (AMIs) for public {op-system-first} are not yet available in the EUSC region. As a workaround, until the AMI publication for {op-system} is extended to the EUSC region, you must edit your `install-config.yaml` file to specify a custom AMI in the `amiID` field.

> * The {aws-short} Security Token Service (STS) is not supported.

Reviewer 3: ```suggestion
* The {aws-short} Security Token Service (STS) is not yet supported.
```

For consistency with other below entries.

Author: Applied this change.

> * Support is not yet provided for installing a cluster into a shared Virtual Private Cloud (VPC).

Reviewer 3: ```suggestion
* Installing a cluster into a shared Virtual Private Cloud (VPC) with a cross-account private hosted zone is not yet supported.
```

Installing into a shared or existing VPC is indeed supported. However, if the user chooses to utilize a private hosted zone from another account (see [EP](https://github.com/openshift/enhancements/blob/master/enhancements/installer/aws-cross-account-dns-zone.md) and [openshift doc](https://docs.redhat.com/en/documentation/openshift_container_platform/4.21/html/installing_on_aws/installer-provisioned-infrastructure#installation-custom-aws-vpc-requirements_installing-restricted-networks-aws-installer-provisioned)), this is not supported.

Sorry, if I was not clear before 😅

Author: Applied this change.

> * Support is not yet provided for bringing your own encryption keys.

Reviewer 3: ```suggestion
```

Technically, it is supported. I have tested this scenario myself, but waiting on storage team to confirm storage testing.

Reviewer 3: cc @rhrmo

Reviewer 4: @tthvo as soon as I manage to get a cluster to start up I will test it and let you know, but it should work imo

Reviewer 4: @tthvo I did a quick test and it works

Reviewer 3: Sounds good, thanks @rhrmo!

Reviewer 3: > Support is not yet provided for bringing your own encryption keys.

@brendan-daly-red-hat can we remove this line as it's confirmed to be supported?

Author: Applied this change.

* `us-west-1` (N. California)
* `us-west-2` (Oregon)

[id="installation-aws-eusc_region{context}"]
== {aws-short} EUSC region
:FeatureName: European Sovereign Cloud (EUSC) region
include::snippets/technology-preview.adoc[leveloffset=+1]

The following {aws-short} European Sovereign Cloud (EUSC) region is supported:

* `eusc-de-east-1` (Brandenburg)

[id="private-clusters-limitations-eusc-aws_{context}"]
== {aws-short} EUSC region limitations
:FeatureName: European Sovereign Cloud (EUSC) region
include::snippets/technology-preview.adoc[leveloffset=+1]

> Installing an {product-title} cluster into the {aws-short} EUSC region helps maximize the sovereignty of your data and satisfy your organization's regulatory requirements. The {aws-short} EUSC is separate and independent from other {aws-short} regions. The infrastructure is located wholly within the European Union (EU). For more information, see "Establishing a European trust service provider for the {aws-short} European Sovereign Cloud".

Reviewer 2: I would move this to the previous section, between lines 58 and 59.

Author: Applied this change.

The following list outlines the limitations that apply to installing an {product-title} cluster into the {aws-short} EUSC region:

* Only one region, `eusc-de-east-1`, and two zones in that region are available.

* The Amazon Machine Images (AMIs) for public {op-system-first} are not yet available in the EUSC region. As a workaround, until the AMI publication for {op-system} is extended to the EUSC region, you must edit your `install-config.yaml` file to specify a custom AMI in the `amiID` field.

* The {aws-short} Security Token Service (STS) is not supported.

* Support is not yet provided for installing a cluster into a shared Virtual Private Cloud (VPC).

> * Support is not yet provided for bringing your own encryption keys.

Reviewer 2: I'd move this into an `[IMPORTANT]` admonition in the previous section, perhaps after you list the Brandenburg region name.

Author: Applied this change.
